src/styles/sidebar/components/menu-item.scss

```diff
@@ -3,96 +3,91 @@
 @use "../../variables" as var;
 
 $menu-item-padding: 10px;
+$item-height: 40px;
 
-.menu-item {
-  @include focus.outline-on-keyboard-focus;
+// The container for the clickable area of the menu item which triggers an
+// action or link. For menu items without submenus, this covers the full area of
+// the menu item. For menu items with submenus, this covers the full area
+// except for the toggle button that opens the submenu.
```

Comment:
I wasn't initially completely sure which CSS this comment referred to. I think it would have been more obvious if the `a.menu-item` rule was nested under `.menu-item`, but AFAIK it isn't possible to do this with SASS (`a&:hover` isn't allowed for example as  the `&` parent selector must appear at the start of the selector).

Let's leave as-is for the moment.